method-discover.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="method-discover_{context}"]
+= Discover phase
+
+.Red Hat migration methodology: Discover phase
+image::RHAMT_AMM_Methodology_446947_0617_ECE_Discover.png[Red Hat Migration Methodology: Discover]
+
+The _Discover_ phase is when you gather all stakeholders and decision-makers together to understand the current state and business drivers, and to determine the migration or modernization needs.
```

Comment:
@HagayVider1 
I would suggest:
"In the _Discover_ phase of migration, all the stakeholders and decision makers of the organization assess the current state and business drivers, and determine the migration or modernization needs."